test/Conversion/triton_to_tritongpu.mlir

```diff
@@ -33,10 +33,10 @@ tt.func @load_ops(%ptr: !tt.ptr<f32> {tt.divisibility = 16 : i32}) {
 tt.func @reduce_ops(%ptr: !tt.ptr<f32> {tt.divisibility = 16 : i32}) {
   // Test if the total number of threadsPerWarp is 64
   // Test if the total number of warps is 2
-  // CHECK: #[[blocked0:.*]] = #triton_gpu.blocked<{sizePerThread = [1, 1], threadsPerWarp = [4, 16], warpsPerCTA = [1, 2], order = [0, 1]}>
-  // CHECK: #[[blocked1:.*]] = #triton_gpu.blocked<{sizePerThread = [1, 1], threadsPerWarp = [8, 8], warpsPerCTA = [1, 2], order = [0, 1]}>
-  // CHECK: #[[blocked2:.*]] = #triton_gpu.blocked<{sizePerThread = [1, 1], threadsPerWarp = [16, 4], warpsPerCTA = [1, 2], order = [0, 1]}>
-  // CHECK: module attributes {"triton_gpu.num-warps" = 2 : i32} {{.*}}
+  // CHECK: #[[blocked0:.*]] = #triton_gpu.blocked<{sizePerThread = [1, 1], threadsPerWarp = [4, 8], warpsPerCTA = [1, 2], order = [0, 1]}>
+  // CHECK: #[[blocked1:.*]] = #triton_gpu.blocked<{sizePerThread = [1, 1], threadsPerWarp = [8, 4], warpsPerCTA = [1, 2], order = [0, 1]}>
+  // CHECK: #[[blocked2:.*]] = #triton_gpu.blocked<{sizePerThread = [1, 1], threadsPerWarp = [16, 2], warpsPerCTA = [1, 2], order = [0, 1]}>
```

Comment:
Same as above, to make this test pass, lines 36 to line 38 needs no changes, and in line 39, "triton_gpu.threads-per-warp" = 64.

Reply:
A temporary solution is to have a private patch for these files on our repo.
A proper solution requires these file support dynamic warp size as well, but this is non-trivial.

Maybe we can address this in a separate PR.

Reply:
I think my point is make what ever changes (or no changes) to make this PR pass the CI build, so we can get it in. Then on navi31, it is acceptable that this individual test failed (we know the reason of the failure, then we make another PR later to address). 

Seems like github does not let the PR merge if CI build failed, so we need to make this test pass. 
If you do not mind, I can change this test so it can pass the CI build.

Reply:
> Seems like github does not let the PR merge if CI build failed

Nope I can get it merged after one approval. I'm just being extra cautious.